pkg/controller/configmap/config.go

```diff
@@ -208,9 +208,13 @@ func Render(configmap *corev1.ConfigMap) ([]*unstructured.Unstructured, error) {
 		}
 	}
 	if haEnabled {
-		renderData.Data[operatortypes.NcpReplicasKey] = operatortypes.NcpHaReplicas
+		renderData.Data[operatortypes.NcpReplicasKey] = ncpReplicas
 	} else {
 		renderData.Data[operatortypes.NcpReplicasKey] = 1
+		if ncpReplicas != 1 {
+			log.Info(fmt.Sprintf("Set nsx-ncp deployment replicas to 1 instead of %d as HA is disabled",
```

Comment:
this looks like a warning log. too bad this logger doesn't have it. maybe we can use its verbosity feature if we stick to this

Reply:
Sure, I'm also thinking to use the log level in our code. How about I update all logs in a following patch.

Reply:
sounds good